File: CJY/BJ/No_2908.js
const rl = require("readline").createInterface({input: process.stdin, output: process.stdout})

rl.on("line", (s) => {
    const arr = s.split(' ');
    let a = "";
    let b = "";
    for (let i = arr[0].length - 1; i >= 0; i--) a += arr[0].charAt(i);
    for (let i = arr[1].length - 1; i >= 0; i--) b += arr[1].charAt(i);
    console.log(a > b ? a : b);
    rl.close();
}).on("close", () => {process.exit();});

File: CJY/BJ/No_2908.java
import java.io.*;

public class No_2908 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] arr = br.readLine().split(" ");
        br.close();

        String a = "";
        String b = "";
        for (int i = arr[0].length() - 1; i >= 0; i--) a += arr[0].charAt(i);
        for (int i = arr[1].length() - 1; i >= 0; i--) b += arr[1].charAt(i);

        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        bw.write(Integer.parseInt(a) > Integer.parseInt(b) ? a : b);

        bw.flush();
        bw.close();
    }
}